### PR TITLE
fix(contentful): Reimplementation of the stemmer-uk without lookbehind regex.

### PR DESCRIPTION
**Pull request by elozano98**
## Description
The stemmer used for Ukrainian had lookbehind regex that was not supported by Safari, so it has been implemented without including that type of unsupported regex.

## Context

Is trying to fix this bug reported by @vanbasten17 in [this issue](https://github.com/axa-group/nlp.js/issues/741).

## Approach taken / Explain the design
The stemmer has been reimplemented in Botonic contentful but replacing the lookbehind regex.

## Testing

The pull request...

- [x] has unit tests
- [x] has integration tests

### CODE REVIEW

**Review by vanbasten17 — APPROVED** [review]

**Review by asastre — APPROVED** [review]

File: packages/botonic-plugin-contentful/src/nlp/stemmers/stemmer-uk.ts [hunk 1]
```diff
@@ -0,0 +1,88 @@
+import Stemmer from '@nlpjs/core/src/stemmer'
```
Comment (dpinol) [inline packages/botonic-plugin-contentful/src/nlp/stemmers/stemmer-uk.ts:1]: in the future, when importing an external file to slightly modify it, please create a commit with the exact external version , and one commit with the change.
In this case, it's enough if you add a comment in the stemmer specifying what was the original line

File: packages/botonic-plugin-contentful/tests/nlp/stemmers/stemmer-uk.test.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { tokenizerPerLocale } from '../../../src/nlp'
+import { StemmerUk } from '../../../src/nlp/stemmers/stemmer-uk'
+
+test.each<any>([
+  ['розмовляючи', ['розмовляюч']],
+  ['говорити', ['говор']],
```
Comment (dpinol) [inline packages/botonic-plugin-contentful/tests/nlp/stemmers/stemmer-uk.test.ts:6]: can you comment which word is using the code which you have fixed?

**Review by dpinol — CHANGES_REQUESTED** [review]
amazing!
can you create a PR for node-nlp, also? or a ticket for it if you have something urgent

**Comment by vanbasten17** [comment]
we already have warned there @dpinol : https://github.com/axa-group/nlp.js/issues/741 👍

**Pull request merged**